packages/liveblocks-yjs/src/index.ts

```diff
@@ -164,15 +160,6 @@ export default class LiveblocksProvider<
   };
 
   private syncDoc = () => {
-    /**
-     * If the connection changes, set the new id, this is used by awareness.
-     * yjs' only requirement for clientID is that it's truly unique and a number.
-     * Liveblock's connectionID satisfies those constraints
-     *  */
-    this.rootDoc.clientID =
-      this.room.getSelf()?.connectionId || this.rootDoc.clientID;
-    this.awareness.clientID = this.rootDoc.clientID; // tell our awareness provider the new ID
```

Comment:
it is now important that we don't set it. Overriding the rootDoc.clientID turned out to be a bad idea. In fact, I'm going to push a change removing the clientID property from the awareness class altogether because it should always use doc.clientID and not be able to be set.